src/main/java/com/wavefront/helpers/Statistics.java

```diff
@@ -71,7 +71,7 @@ public String toJSONString() throws Exception {
     tracesByType.forEach((k, v) -> {
       ObjectNode node = mapper.createObjectNode();
       node.put("Count", v.count);
-      node.put("Percentage", Math.round((double) v.count / tracesSum * 100));
+      node.put("Percentage", (double) v.count / tracesSum * 100);
```

Comment:
Instead of (double) v.count / tracesSum * 100
you can use 100.0 * v.count / tracesSum